builder/builder.go

```diff
@@ -261,25 +276,32 @@ func (b *Builder) storeWithdrawalMsgInEVM(
 	withdrawalMsg *rollupv1.InitiateWithdrawalRequest,
 	ethState *state.StateDB,
 	header *monomer.Header,
-) error {
+) (*big.Int, error) {
```

Comment:
Should add the meaning of this return value to `storeWithdrawalMsgInEVM`'s godoc.